modal/_serialization.py

```diff
@@ -19,6 +19,7 @@ def get_synchronicity_interface(obj) -> Optional[Interface]:
 
 
 def restore_synchronicity_interface(raw_obj, target_interface: Optional[Interface]):
+    print("Unpickling", raw_obj.object_id, "as", target_interface)
```

Comment:
debug print?

Reply:
Ugh, must have accidentally `commit -a`:d it :( Will check so nothing else weird made it in...